examples/kops-new-cluster/base/cluster.yaml

```diff
@@ -10,7 +10,7 @@ spec:
     rbac: {}
```

Comment:
Per your comment on the other PR, we should drop these cluster specs?

Reply:
When 1.24 is out, I can restructure this bit.